test/fluree/db/query/sparql_test.cljc

```diff
@@ -140,6 +140,20 @@
       (is (= [{"@id" "?person", "person:handle" "?handle"}
               {"@id" "?person", "person:favNums" "?num"}
               [:filter ["(> ?num 10)"]]]
+             where)))
+    (let [query "PREFIX psm: <http://srv.ktbl.de/data/psm/>
+                 PREFIX schema: <http://schema.org/>
+                 SELECT ?s ?t ?name
+                 FROM <cookbook/base>
+                 WHERE {
+                   ?s <@type> ?t.
+                   ?s schema:name ?name.
+                   FILTER REGEX(?name, \"^Jon\", \"i\")
+                 }"
+          {:keys [where]} (sparql/->fql query)]
+      (is (= [{"@id" "?s", "@type" "?t"}
+              {"@id" "?s", "schema:name" "?name"}
+              [:filter ["(regex ?name \"^Jon\" \"i\")"]]]
```

Comment:
What is this testing? Just that REGEX filters are translated correctly? Might be good to add a `testing` annotation (or similar) to that effect. Looks like the `psm` prefix is unused, is that intentional?